server/src/application/application.service.ts

```diff
@@ -11,6 +13,20 @@ export class ApplicationService {
     private applicationRepository: Repository<Application>
   ) {}
 
+  async getApplications(query: GetApplicationsDto): Promise<PaginationRO<Application>> | null {
+    const queryBuilder = this.applicationRepository.createQueryBuilder('app');
+
+    if (query.facilityName)
+      queryBuilder.andWhere('app.facilityName like :facilityName', {
```

Comment:
like is a case-sensitive check. Use ILIKE!

Reply:
Thanks, had no idea, changing.